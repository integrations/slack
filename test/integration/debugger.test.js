const { models } = require('.');

const { GitHubUser } = models;

describe('Debugger', () => {
  test('should re-throw errors', async () => {
    await expect(GitHubUser.findOrCreate({ where: { nonExistingColumn: 'anything' } })).rejects.toMatchSnapshot();
  });
});
