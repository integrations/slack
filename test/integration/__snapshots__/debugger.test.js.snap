// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debugger should re-throw errors 1`] = `[SequelizeDatabaseError: column GitHubUser.nonExistingColumn does not exist]`;

exports[`Debugger should re-throw errors 2`] = `Set {}`;

exports[`Debugger should re-throw errors 3`] = `Map {}`;
